Modules/_sqlite/util.c

```diff
@@ -135,21 +135,22 @@ set_error_from_code(pysqlite_state *state, int code)
 /**
  * Checks the SQLite error code and sets the appropriate DB-API exception.
  */
-void
+int
```

Comment:
In `blob.c`, should `blob_seterror` assert that `set_error_from_db` didn't return `SQLITE_OK`?
Same for when `connection.c` and the files changed here call `blob_seterror` without checking the result.

Reply:
Oh I missed the blob stuff. I think we should also assert this indeed.

Reply:
Also, I think there was an API misuse here. We pass `rc` but we never use it. I think we were meant to use `set_error_from_code` instead.